Automation/createresource.ps1

```diff
@@ -0,0 +1,12 @@
+$resourceGroupName = Read-Host -Prompt "Enter the Resource Group name"
+$location = Read-Host -Prompt "Enter the location (i.e. centralus)"
+$storageAccountName = Read-Host -Prompt "Enter the storage account name"
```

Comment:
This could be better if taken in as parameters instead of prompts so whoever runs it can just do it in one line.